src/MergeContext.js

```diff
@@ -668,7 +668,9 @@ class PullRequest {
             const prMergeSha = await GH.getReference(this._mergePath());
             const prCommit = await GH.getCommit(prMergeSha);
             // whether the PR branch has not changed
-            if (this._stagedCommit.tree.sha === prCommit.tree.sha) {
+            const stagedCommitDate = new Date(this._stagedCommit.committer.date);
+            const prCommitDate = new Date(prCommit.committer.date);
+            if (stagedCommitDate > prCommitDate) {
```

Comment:
I doubt this matters in practice, but to support cases where the GitHub creates merge commits using the last PR branch commit date _and_ to avoid an implication that we do not support instant merge commit creation, can we treat same dates as fresh?

```suggestion
            if (stagedCommitDate >= prCommitDate) {
```

Reply:
Agreed.